core/src/main/java/jenkins/install/SetupWizard.java

```diff
@@ -54,6 +56,7 @@
  * 
  * @since 2.0
  */
+@Restricted(NoExternalUse.class) // use by Jelly
 public class SetupWizard {
```

Comment:
I'm not convinced that SetupWizard can be used in Jelly only.
In the future plugin instances may hook on the wizard during the installation process in order to contribute some installation steps.